src/marketplace/ListingsView.js

```diff
@@ -196,26 +199,40 @@ class ListingsView extends React.Component<Props, State> {
 			response => response.json().then(data => {
 				if (response.status !== 201) {
 					console.log(response.status);
-					console.log(data.msg);
+					toast(data.msg);
 					console.log(data);
 				}
 				else {
-					console.log("success");
+					toast("Item Bought");
```

Comment:
I think toast.success is usually more common for these types of things. If you really think the default is better, I'm okay with it, but just make sure you know that the success one exists (because it was made for success messages after all)